src/main/java/mate/academy/controller/AuthenticationController.java

```diff
@@ -36,9 +36,10 @@ public ResponseEntity<Object> login(@RequestBody @Valid UserLoginDto userLoginDt
             throws AuthenticationException {
         User user = authenticationService.login(userLoginDto.getLogin(),
                 userLoginDto.getPassword());
-        String token = jwtTokenProvider.createToken(user.getEmail(), user.getRoles().stream()
+        List<String> roles = user.getRoles().stream()
                 .map(role -> role.getRoleName().name())
-                .collect(Collectors.toList()));
+                .toList();
+        String token = jwtTokenProvider.createToken(user.getEmail(), roles);
```

Comment:
after the merge, this PR will our students get a merge conflict?

Reply:
yes

Reply:
if they will change this file